packages/admin/admin/src/common/toolbar/Toolbar.tsx

```diff
@@ -5,17 +5,22 @@ import * as React from "react";
 import { createComponentSlot } from "../../helpers/createComponentSlot";
 import { ThemedComponentBaseProps } from "../../helpers/ThemedComponentBaseProps";
 import { MasterLayoutContext } from "../../mui/MasterLayoutContext";
+import { ToolbarBreadcrumbs } from "./ToolbarBreadcrumbs";
 
-export type ToolbarClassKey = "root" | "muiToolbar" | "mainContentContainer";
+export type ToolbarClassKey = "root" | "topBar" | "muiToolbar" | "mainContentContainer";
 
 export interface ToolbarProps
     extends ThemedComponentBaseProps<{
         root: typeof Paper;
         muiToolbar: typeof MuiToolbar;
         mainContentContainer: "div";
+        topBar: "div";
     }> {
     elevation?: number;
     children?: React.ReactNode;
+    scopeIndicator?: React.ReactNode;
+    hideTopBar?: boolean;
+    hideBottomBar?: boolean;
```

Comment:
Why do we need this two props? Can't this be derived from the passed content instead?

Reply:
`hideBottomBar` can typically be derived from the children.

My intention for `hideTopBar` was to use the same `Toolbar` for the `DataGridToolbar`. We can't automatically determine if the top bar should be shown. Even if no scope indicator is passed, the breadcrumbs could still be desired. 
We could also achieve this by making the `BottomBar` an independent component and use it for the `DataGridToolbar`